torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -1379,6 +1385,12 @@ def is_all_inputs_connected_to_quantized_op(input_nodes):
                 if not is_all_inputs_connected_to_quantized_op(input_nodes_to_check):
                     return
                 self._annotate_cat(node, quantization_config)
+            elif (
+                node.target is torch.ops.aten.flatten.using_ints
+                and len(node.users) > 0
+                and not is_any_users_connected_to_quantizable_op(node.users)
```

Comment:
nit: `not any([user for user in list(node.users.keys()) if user.target in quantizable_ops])`

Reply:
Please also add a comment for the special recipe of `flatten` here.